hippunfold/config/snakebids.yml

```diff
@@ -221,6 +221,10 @@ parse_args:
     nargs: '+'
     help: 'Hemisphere(s) to process (default: %(default)s)'
 
+  --NoIsovolume:
```

Comment:
couple comments:
 - we should use lowercase for argument names
 - but I think might be better to instead have an option called --laminar_coords_method, that can be either "laplace" or "isovolume" (can add others if implemented too), and can default to "isovolume"

Reply:
fixed